ipn/ipnlocal/local.go

```diff
@@ -5260,18 +5266,15 @@ func (b *LocalBackend) applyPrefsToHostinfoLocked(hi *tailcfg.Hostinfo, prefs ip
 
 	hi.ServicesHash = b.vipServiceHash(b.vipServicesFromPrefsLocked(prefs))
 
-	// The Hostinfo.WantIngress field tells control whether this node wants to
-	// be wired up for ingress connections. If harmless if it's accidentally
-	// true; the actual policy is controlled in tailscaled by ServeConfig. But
-	// if this is accidentally false, then control may not configure DNS
-	// properly. This exists as an optimization to control to program fewer DNS
-	// records that have ingress enabled but are not actually being used.
-	// TODO(irbekrm): once control knows that if hostinfo.IngressEnabled is true,
-	// then wireIngress can be considered true, don't send wireIngress in that case.
-	hi.WireIngress = b.wantIngressLocked()
 	// The Hostinfo.IngressEnabled field is used to communicate to control whether
-	// the funnel is actually enabled.
+	// the node has funnel enabled.
 	hi.IngressEnabled = b.hasIngressEnabledLocked()
+	// The Hostinfo.WantIngress field tells control whether the user intends to use funnel with this node even
+	// though it is not currently enabled. This is an optimization to control- Funnel requires creation of DNS
+	// records and because DNS propagation can take time, we want to ensure that the records exist for any node that
+	// intends to use funnel even if it's not enabled. If hi.IngressEnabled is true, control knows that DNS records
+	// are needed, so we can save bandwidth and not send WireIngress.
```

Comment:
My Emacs (`fill-paragraph`) and VS Code (https://marketplace.visualstudio.com/items?itemName=stkb.rewrap) paragraph wrapping both wrap at more like 80 columns. Not sure what you're using, but maybe you can adjust it down a bit? We're not super strict about column width in our tree, but keeping it short-ish is more friendly to people who like to browse with lots of narrow windows.